cluster/charts/universal-crossplane/values.yaml.tmpl

```diff
@@ -81,10 +80,10 @@ packageCache:
 resourcesRBACManager:
   limits:
     cpu: 100m
-    memory: 768Mi
+    memory: 512Mi
   requests:
     cpu: 100m
-    memory: 512Mi
+    memory: 256Mi
```

Comment:
With the [recent improvements in the RBAC manager](https://github.com/crossplane/crossplane/pull/4031), I don't think we would need extra resources here. Similar to the above, if required for some specific use case, it could always be overridden as part of the helm installation. 

Introduced with https://github.com/upbound/universal-crossplane/pull/265
/cc @negz @avalanche123